### PR TITLE
[5.8] Fix validate method

### LINKED ISSUE #25736: [FormRequest] validated() method causes the validation to be called again.
- Laravel Version: 5.7.5
- PHP Version: 7.2.9
- Database Driver & Version: MySQL 5.7.23

### Description:
The problem is getting the retrieve the validated input data with method `validated()`. When it is called, it re-initiates the request validation. It seems like 941c8c7be2ad4441cdba12053c15f5dc46980707 commit breaks this.

### Steps To Reproduce:
1. Controller:
```
use App\Http\Requests\ClientStoreRequest;

class ClientController extends Controller
{
    public function store(ClientStoreRequest $request)
    {
        $this->clientService->createClient($request->validated());

        return redirect()->route('app.clients')->with('status', 'Client was successfully created!');
    }
}
```
2. FormRequest:
```
use Illuminate\Foundation\Http\FormRequest;
use Illuminate\Validation\Rule;

class ClientStoreRequest extends FormRequest
{
    public function rules()
    {
        return [
	    'full_name' => 'required|string|max:50',
	    'phone' => [
	        'required',
	        'max:50',
	        'regex:/^(\+[0-9]+)$/',
	        Rule::unique('clients')->where(function($query){
	            return $query->where('account_id', auth()->user()->account_id);
	        })
	    ],
	    'email' => 'nullable|email|max:50',
	    'info' => 'max:200'
        ];
    }
}
```
3. Debugbar:
Since the rules have a check in the database, we can see a duplicated query to the database, which means that the method rules() was called twice.
debugbar_546654652654

### PR DESCRIPTION
**Pull request by ttsuru**
Issue fix #25736, #25750.
From #26417

941c8c7 cause when Illuminate\Foundation\Http\FormRequest::validated is called. validated again.

### CODE REVIEW

File: src/Illuminate/Contracts/Validation/Validator.php [hunk 1]
```diff
@@ -13,6 +13,13 @@
      */
     public function validate();
 
+    /**
+     * Return validated value.
```
Comment (jmarcher) [inline src/Illuminate/Contracts/Validation/Validator.php]: Change to: "Return validated values." it should be plural

File: src/Illuminate/Validation/Validator.php [hunk 2]
```diff
@@ -306,6 +306,22 @@ public function validate()
             throw new ValidationException($this);
         }
 
+        return $this->validated();
+    }
+
+    /**
+     * Return validated value.
```
Comment (jmarcher) [inline src/Illuminate/Validation/Validator.php]: Change to: "Return validated values." it should be plural

**Review by jmarcher — CHANGES_REQUESTED** [review]

File: src/Illuminate/Validation/Validator.php (hunk 2 above)
Comment (ttsuru) [reply src/Illuminate/Validation/Validator.php]: @jmarcher Thanks for review. Fixed.

File: src/Illuminate/Contracts/Validation/Validator.php (hunk 1 above)
Comment (ttsuru) [reply src/Illuminate/Contracts/Validation/Validator.php]: @jmarcher Thanks for review. Fixed.

**Review by jmarcher — APPROVED** [review]

**Pull request merged**